packages/flutter_riverpod/test/framework_test.dart

```diff
@@ -247,8 +268,7 @@ void main() {
     expect(error, isNotNull);
   });
 
-  testWidgets('ref.watch within a build method can flush providers',
-      (tester) async {
+  testWidgets('ref.watch within a build method can flush providers', (tester) async {
```

Comment:
Please don't change the line length :P

Reply:
hmm, autoformatting from IntelliJ I think